src/app.ts

```diff
@@ -36,8 +55,53 @@ export default async function newApp(): Promise<Express> {
   app.use(bodyParser.json());
   app.use(bodyParser.urlencoded({ extended: true }));
 
+  app.use(passport.initialize());
+
   registerRoutes(app);
 
+  // SAML settings.
+  const SAML_CALLBACK_URL =
+    process.env.SAML_CALLBACK_URL ||
+    "http://io-onboarding-backend:3000/assertion-consumer-service";
```

Comment:
defaults should always be production configs - or else (better), fail if settings is not defined